lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -99,7 +99,7 @@ function dosomething_campaign_load($node, $public = FALSE) {
   $campaign->title = $node->title;
```

Comment:
can we get rid of this line then ` $wrapper = dosomething_helpers_node_metadata_wrapper($node);`